app/pages/search/search-controls/SearchControls.js

```diff
@@ -53,6 +64,16 @@ class SearchControls extends Component {
               value={this.state.date}
             />
           </FormGroup>
+          <Checkbox
+            className="is-favorite-control-group"
+            controlId="is-favorite-control-group"
+            onChange={event =>
+              this.handleChange({ isFavorite: event.target.checked.toString() })
+            }
+            checked={this.state.isFavorite === 'true'}
+          >
+            <ControlLabel>Näytä vain suosikit</ControlLabel>
```

Comment:
- [ ] The "label" for checkbox should be just text so remove the `ControlLabel` component:
```javascript
<Checkbox { ... }>
  Näytä vain suosikit
</Checkbox>
```